src/lib/components/CartoonCard.svelte

```diff
@@ -1,13 +1,17 @@
 <script>
-    import src from '$lib/cartoon-card-placeholder.svg';
+    export let src = '$lib/lino-sablay.svg';
```

Comment:
I think it's better if we explicitly require the caller to provide the `src` rather than defaulting and assuming that `lino-sablay.svg` is what the caller meant to put.
```suggestion
    // eslint-disable-next-line init-declarations
    export let src: string;
```